sdk/dataprotection/Azure.ResourceManager.DataProtectionBackup/src/Generated/Models/BackupInstancePolicyInfo.cs

```diff
@@ -45,6 +45,8 @@ internal BackupInstancePolicyInfo(ResourceIdentifier policyId, string policyVers
         /// Gets or sets the DataStore Parameters
         /// Please note <see cref="DataStoreSettings"/> is the base class. According to the scenario, a derived class of the base class might need to be assigned here, or this property needs to be casted to one of the possible derived classes.
         /// The available derived classes include <see cref="OperationalDataStoreSettings"/>.
+        /// Please note <see cref="DataStoreSettings"/> is the base class. According to the scenario, a derived class of the base class might need to be assigned here, or this property needs to be casted to one of the possible derived classes.
```

Comment:
why are we doubling the comment here?

Reply:
Well I did not find the root cause for this yet.
This only happens once and it is doubling something instead of missing something therefore I am planning to finish this feature with this issue and we fix this later
what do you think?

Reply:
OK, this has been fixed.